<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RGR GUI2022</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="pageMiskkomunhoz">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>45</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of Miskkomunhoz</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSysName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Miskkomunhoz</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageZhek">
     <widget class="QLineEdit" name="lineEdit_Name_Zhek">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ZhEK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of ZhEK</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageProfession">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of profession</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Name_Profession">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Profession</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageWorker">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name of worker</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>95</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Name_Worker">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>45</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>95</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>25</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="store">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Store</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Update">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Query_edit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Find_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Find by fist letter</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_find">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>31</y>
      <width>256</width>
      <height>241</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="box_Query_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Query_2">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>40</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average age</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStore"/>
    <addaction name="actionRestore"/>
   </widget>
   <widget class="QMenu" name="menuTree_operations">
    <property name="title">
     <string>Tree operations</string>
    </property>
    <addaction name="actionCreate_root"/>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuQuery_operations">
    <property name="title">
     <string>Query operations</string>
    </property>
    <addaction name="actionQuery1"/>
    <addaction name="actionQuery2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTree_operations"/>
   <addaction name="menuQuery_operations"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStore">
   <property name="text">
    <string>Store</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionCreate_root">
   <property name="text">
    <string>Create root</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionQuery1">
   <property name="text">
    <string>Query 1</string>
   </property>
  </action>
  <action name="actionQuery2">
   <property name="text">
    <string>Query 2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
